scripts/travis/travis_build_and_test.sh

```diff
@@ -6,7 +6,7 @@ set -e
 MAKE="make --jobs=$NUM_THREADS --keep-going"
 
 if $WITH_CMAKE; then
-  mkdir build
+  mkdir -p build
```

Comment:
Is this so you can run the script locally? I don't think we need to check that into GitHub.